docs/getting_started/install_on_ibexa_engage.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Install and configure Ibexa Engage.
+edition: experience
+---
+
+# Install on Ibexa Engage
+
+[[= product_name_engage =]] is a data collection tool. It enables you to engage your audiences by using the [Qualifio](https://qualifio.com/) tools.
+You can use interactive content to gather valuable data, for example, customer personal data or recent orders list, and create connections.
+
+## Prepare configuration files
+
+First, run the `composer ibexa:setup` command:
+
+``` bash
+composer require ibexa/connector-qualifio
```

Comment:
it’s not required anymore it’s part of v4.6 Experience
Please double check with backend team @Steveb-p @adamwojs

Reply:
Updated as discussed with @adamwojs.